README.md

```diff
@@ -176,11 +209,11 @@ Deployed:
 | Counter  | 0x2e11Ed82f5eC165AB8Ce3cC094f025Fe7527F4D1                        |
 
 Declared:
-
+<!-- markdownlint-disable MD013 -->
 | Contract                 | Class hash                                                        |
 | ------------------------ | ----------------------------------------------------------------- |
 | Proxy account class hash | 0x3010a53967fa04842bcbcb6de8817101f047ef0d074b3eacbe714a3fc42a2eb |
-
+<!-- markdownlint-enable MD013 -->
```

Comment:
Couldn't fix this one so I added an ignore for this portion

Reply:
anyway I'll clean the README once this is merged, this info is not accurate anymore!